uqcsbot/scripts/dominos.py

```diff
@@ -7,40 +7,43 @@
 from uqcsbot.utils.command_utils import loading_status, UsageSyntaxException
 import requests
 
-MAX_COUPONS = 10 # Prevents abuse
+MAX_COUPONS = 10  # Prevents abuse
 COUPONESE_DOMINOS_URL = 'https://www.couponese.com/store/dominos.com.au/'
 
+
 class Coupon:
     def __init__(self, code: str, expiry_date: str, description: str) -> None:
         self.code = code
         self.expiry_date = expiry_date
         self.description = description
-    
+
     def is_valid(self) -> bool:
         try:
             expiry_date = datetime.strptime(self.expiry_date, '%Y-%m-%d')
             now = datetime.now()
-            return expiry_date.year >= now.year and expiry_date.month >= now.month and\
-                expiry_date.day >= now.day
-        except:
+            return (expiry_date.year >= now.year and expiry_date.month >= now.month and
+                    expiry_date.day >= now.day)
+        except Exception:
```

Comment:
I would certainly prefer more specific exceptions

Reply:
Any suggestions?

Reply:
Pretty sure you're only expecting ValueErrors from strptime.

Reply:
Fixed.